wdl-analysis/src/diagnostics.rs

```diff
@@ -657,7 +657,7 @@ pub fn unknown_function(name: &str, span: Span) -> Diagnostic {
 /// Creates an "unsupported function" diagnostic.
 pub fn unsupported_function(minimum: SupportedVersion, name: &str, span: Span) -> Diagnostic {
     Diagnostic::error(format!(
-        "function `{name}` requires a minimum WDL version of {minimum}"
+        "this use of function `{name}` requires a minimum WDL version of {minimum}"
```

Comment:
FYI, the language here was changed as it's possible that the function itself has a supported minimum version, but the way the function was called required a newer version of WDL to be used (i.e. you're calling an overload that exists in a later WDL version).